bangazonapi/views/profile.py

```diff
@@ -233,17 +233,16 @@ def cart(self, request):
             """
 
             try:
-                open_order = Order.objects.get(customer=current_user)
-                print(open_order)
+                open_order = Order.objects.filter(customer=current_user).last()
             except Order.DoesNotExist as ex:
                 open_order = Order()
                 open_order.created_date = datetime.datetime.now()
                 open_order.customer = current_user
                 open_order.save()
 
             line_item = OrderProduct()
-            line_item.product = Product.objects.get(
-                pk=request.data["product_id"])
+            line_item.product = Product.objects.filter(
+                pk=request.data["product_id"])[0]
```

Comment:
The code kept failing at this line without this fix.

Reply:
The assertion error keeps coming up, whenever I use .get here. I'm still unsure why this is happening in line 245 in particular.

Reply:
Resolved in latest commit!